signer/src/message.rs

```diff
@@ -164,6 +166,10 @@ pub struct SweepTransactionInfo {
 impl SweepTransactionInfo {
     /// Creates a [`SweepTransactionInfo`] from an [`UnsignedTransaction`] and a
     /// Bitcoin block hash.
+    ///
+    /// **Note that it is important that this function is called after the
+    /// transaction has been signed**, as otherwise the `vbytes` size will not
+    /// be correct.
```

Comment:
When creating an `UnsignedTransaction` we compute what the `vsize` would be if it were signed, but then throw that value away since we didn't need it at the time. We could keep it around if we want to.

Reply:
Ah good to know, I'll make that change

Reply:
Updated in [a399760](https://github.com/stacks-network/sbtc/pull/810/commits/a39976075a79f2c064b214d368485b787a1139e0)